github_hooks/db/migrate/20251119104017_add_organization_created_index_at_jobs_table.rb

```diff
@@ -0,0 +1,11 @@
+class AddOrganizationCreatedIndexAtJobsTable < ActiveRecord::Migration[6.1]
+  disable_ddl_transaction!
+
+  def change
+    add_index :jobs,
```

Comment:
@loadez please check could the extended time required for the CONCURRENTLY index build lead to liveness/readiness Probe failure (as Puma is not running), resulting in Kubernetes terminating the pod before completion? Should we run this as a dedicated Job?

Reply:
I validated with the team and we will run this migration with extended timeouts.